models/palette_model.py

```diff
@@ -80,6 +74,84 @@ def modify_commandline_options(parser, is_train=True):
             default=[0, 255 * 3],
             help="range for Canny thresholds",
         )
+
+        parser.add_argument(
```

Comment:
My appreciation is that all these options should not be `_palette_` but `data_aug`, and therefore moved out of here, into `train_options.py`. CC @pnsuau @royale same for canny later on. This refactoring could start in this PR, and canny be moved later on.